data/json/recipes/weapon/magazines.json

```diff
@@ -9,8 +9,26 @@
     "skills_required": [ "gun", 2 ],
     "time": "20 m",
     "book_learn": [ [ "recipe_bullets", 2 ], [ "mag_rifle", 3 ] ],
+    "tools": [
+      [
+        [ "3006_casing", -1 ],
+        [ "270win_casing", -1 ],
+        [ "3006", -1 ],
+        [ "3006_casing", -1 ],
```

Comment:
This entry appears twice here.

Perhaps make those into requirements, by the way? Would make later maintenance much easier.